test/type-error-2.ktn:3:1: error: cannot match scalar type [t1] with int
test/type-error-2.ktn:3:9: note: int (from baz) is from here
test/type-error-2.ktn:4:12: note: [t1] (output of __add_vector) is from here
